examples/runners/wgpu/src/lib.rs

```diff
@@ -70,11 +70,14 @@ fn shader_module(shader: RustGPUShader) -> wgpu::ShaderModuleDescriptor<'static>
         // under cargo by setting these environment variables.
         std::env::set_var("OUT_DIR", env!("OUT_DIR"));
         std::env::set_var("PROFILE", env!("PROFILE"));
-        let crate_name = match shader {
-            RustGPUShader::Simplest => "sky-shader",
-            RustGPUShader::Sky => "simplest-shader",
-            RustGPUShader::Compute => "compute-shader",
-            RustGPUShader::Mouse => "mouse-shader",
+        let (crate_name, capabilities): (_, &[Capability]) = match shader {
+            RustGPUShader::Simplest => ("sky-shader", &[]),
+            RustGPUShader::Sky => ("simplest-shader", &[]),
+            RustGPUShader::Compute => ("compute-shader", &[]),
+            RustGPUShader::Mouse => (
+                "mouse-shader",
+                &[Capability::Int8, Capability::Int16, Capability::Int64],
```

Comment:
Oof, the duplication here isn't great. There's no easy way to share this information, is there?

Reply:
My only thought was to make a common file and `include!()` it from both (or `#[path = "..."] mod blah;` or whatever), but I'm not sure if that makes it better or worse.